docs/tutorials/q32d14.ipynb

```diff
@@ -51,10 +51,10 @@
     "    <a target=\"_blank\" href=\"https://quantumai.google/qsim/tutorials/q32d14\"><img src=\"https://quantumai.google/site-assets/images/buttons/quantumai_logo_1x.png\" />View on QuantumAI</a>\n",
     "  </td>\n",
     "  <td>\n",
-    "    <a target=\"_blank\" href=\"https://colab.research.google.com/github/quantumlib/qsim/blob/master/docs/tutorials/q32d14.ipynb\"><img src=\"https://quantumai.google/site-assets/images/buttons/colab_logo_1x.png\" />Run in Google Colab</a>\n",
+"    <a target=\\"_blank\\" href=\\"https://colab.research.google.com/github/quantumlib/qsim/blob/main/docs/tutorials/q32d14.ipynb\\"><img src=\\"https://quantumai.google/site-assets/images/buttons/colab_logo_1x.png\\" />Run in Google Colab</a>\n",
```

Comment:
The JSON syntax is broken here, please fix.  As a quick check you can run
`jq . docs/tutorials/q32d14.ipynb >/dev/null` to verify the content is parseable.